spec/support/acceptance/meterpreter.rb

```diff
@@ -14,21 +14,21 @@ def self.current_platform
     end
   end
 
-  # Allows restricting the tests of a specific Meterpreter's test suite with the METERPRETER environment variable
-  # @return [TrueClass, FalseClass] True if the given Meterpreter should be run, false otherwise.
-  def self.run_meterpreter?(meterpreter_config)
-    return true if ENV['METERPRETER'].blank?
+  # Allows restricting the tests of a specific session's test suite with the SESSION environment variable
+  # @return [TrueClass, FalseClass] True if the given session should be run, false otherwise.
+  def self.run_session?(session_config)
```

Comment:
I think we still want unique identifiers here for environment variables, for the scenario of running the test suite all at once instead of on a per-file basis 🤔 

I think we should either revert `run_meterpreter?` and `run_meterpreter_module_test?`, or update it to take a parameter of the session type that's expected

Reply:
Happy to jump on a call if you disagree with that approach, i.e. if I'm missing an edgecase or such and you'd prefer to follow a different pattern